tests/PostTest.php

```diff
@@ -176,4 +176,19 @@ public function testPostRelationConnections()
 
         $this->assertEquals('no_prefix', $post->author->getConnectionName());
     }
+
+    public function testMenuItemActsAsObjectFacadeForTitleAndSlug()
```

Comment:
What type of menu testing are you making here? It's better to make menu tests inside `tests/MenuTest.php`.